src/js/edu-benefits/5490/config/form.js

```diff
@@ -498,6 +500,15 @@ const formConfig = {
             'educationalCounseling'
           ],
           required: ['educationType']
+        }), {
+          // Rephrase the question for facility name in educationProgram
```

Comment:
Ah, gotcha. I think when we switch the 1995 to use educationProgram, we should make this one the default, since I think it's more common that previous education.